### DISCUSSION TITLE
Action Button action to launch Databrowser

### DISCUSSION THREAD

**Discussion by georgweiss** [ControlSystemStudio/phoebus#3220] [Ideas]
I have a user request to add another action type to the Action Button: launch Databrowser with plt file. 

Any thoughts?

**Comment by shroffk** [comment]
We can do this with the "open file" option
![image](https://github.com/user-attachments/assets/2ec7431b-95e1-4f78-8971-024d618e0b02)

**Comment by shroffk** [comment]
TODO: 

I would like to expand the functionality so that we can open "layouts" and other app resources.

Not sure if we should create a new actions type "open resources" or try to build on this one
